skimage/transform/_warps.py

```diff
@@ -106,17 +107,19 @@ def resize(image, output_shape, order=None, mode='reflect', cval=0, clip=True,
                          "dimensions")
 
     if anti_aliasing is None:
-        anti_aliasing = not image.dtype == bool
+        anti_aliasing = not input_type == bool
```

Comment:
```suggestion
        anti_aliasing = (not input_type == bool and
                         np.any(output_shape < input_shape))
```

Reply:
@abouysso, I forgot about the suggestion above, can you please implement it?

Reply:
Done !

Reply:
Actually this proposition does not seem to work : comparing two tuples always returns a boolean and so np.any() seems to have no effect. My guess is *that* should do what was intended:

```python
anti_aliasing = (not input_type == bool and any(x < y for x,y in zip(output_shape, input_shape)))
```